pkg/provider/provider.go

```diff
@@ -68,24 +68,16 @@ type TestEnvironment struct { // rename this with testTarget
 	AbnormalEvents []*Event
 
 	// Pod Groupings
-	Pods                     []*Pod                 `json:"testPods"`
-	DebugPods                map[string]*corev1.Pod // map from nodename to debugPod
-	GuaranteedPods           []*Pod
-	NonGuaranteedPods        []*Pod
-	AffinityRequiredPods     []*Pod
-	AntiAffinityRequiredPods []*Pod
-	HugepagesPods            []*Pod
-	AllPods                  []*Pod `json:"AllPods"`
+	Pods          []*Pod                 `json:"testPods"`
+	DebugPods     map[string]*corev1.Pod // map from nodename to debugPod
+	HugepagesPods []*Pod
```

Comment:
Can't we do another getter in filters_test.go for this HugepagesPods list?

Reply:
Yes I can add that.